angularjs-portal-frame/src/main/webapp/samples/welcome.json

```diff
@@ -0,0 +1,11 @@
+{ 
+   "id" : 1,
+   "title" : "",
+   "img" : "",
+   "learnMoreURL" : "",
+	 "startYear" : 2015,
+	 "startMonth" : 7,
+	 "startDay" : 11,
+	 "accessibilityText" : "",
+	 "buttonText" : ""
+}
```

Comment:
Thoughts on just making this an array instead of an object? So `/frame` can just be `[]`.

Then in the controller you can just check the length, and if the length is > 0 then go through the date logic.